inc/knowbaseitem_category.class.php

```diff
@@ -0,0 +1,285 @@
+<?php
+/**
+ * ---------------------------------------------------------------------
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2015-2021 Teclib' and contributors.
+ *
+ * http://glpi-project.org
+ *
+ * based on GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2003-2014 by the INDEPNET Development Team.
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * GLPI is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * GLPI is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ * ---------------------------------------------------------------------
+ */
+
+if (!defined('GLPI_ROOT')) {
+   die("Sorry. You can't access this file directly");
+}
+
+/**
+ *  Class KnowbaseItem_Category
+ *
+ *  @author Romain Biot <rbiot@teclib.com>
+ *
+ *  @since 9.5.6
+ */
+class KnowbaseItem_Category extends CommonDBRelation {
```

Comment:
I am afraid people will confuse `KnowbaseItemCategory` and `KnowbaseItem_Category`.
According to GLPI conventions, we should name this class `KnowbaseItem_KnowbaseItemCategory`. Seem a long class name, but I guess it is preferable. @trasher @orthagh ?

Reply:
agree